CMakeLists.txt

```diff
@@ -602,7 +602,12 @@ elseif(EMSCRIPTEN)
   target_sources(lovr PRIVATE src/core/os_web.c)
   target_compile_definitions(lovr PRIVATE LOVR_WEBGL)
 elseif(ANDROID)
-  target_link_libraries(lovr log EGL GLESv3 OpenSLES android dl)
+  target_sources(lovr PRIVATE src/core/os_android.c)
```

Comment:
Is this line adding os_android here by mistake?  That file can't be used with pico